package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-katex",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
you could undo this. using this fork requires referencing this git repo directly - the version in package.json is only important for publishing to npm. probably valuable to keep this version the same as what react-katex was at the point I forked this.

Reply:
Makes sense. So at this point, Course Hero does have a org set up on Github for open source. Unless you're using Katex a lot and updating this anyway, we could take a little off your plate and transfer this repo to the CH open source account.

Reply:
I'm not using it at all. I can transfer to the CH org if you like.

Really, someone needs to get this change to land upstream: https://github.com/talyssonoc/react-katex/pull/42